arch/xtensa/src/common/xtensa_int_handlers.S

```diff
@@ -68,6 +68,19 @@
 #include "chip_macros.h"
 #include "xtensa_timer.h"
 
+#if !defined(CONFIG_SMP) && CONFIG_ARCH_INTERRUPTSTACK > 15
+ .data
+ .align     16
```

Comment:
For Xtensa architecture, does .align 16 means 16-byte alignment?
For ARM architecture, .align means 2^n but it confuses me. So I will refactor them.

However, we can use .balign instead of .align.
Please see the following URL.
https://developer.arm.com/documentation/dui0742/c/migrating-arm-syntax-assembly-code-to-gnu-syntax/alignment

Reply:
Yes, it means 16-byte alignement. (ref. https://sourceware.org/binutils/docs-2.25/as/Align.html)
Here is the relevant text:

> The way the required alignment is specified varies from system to system. For ....and xtensa, the first expression is the alignment request in bytes.

There is also a clearer example [here](https://github.com/jcmvbkbc/linux-xtensa/blob/f75aef392f869018f78cfedf3c320a6b3fcfda6b/arch/xtensa/kernel/vectors.S#L663).